### PR TITLE
Fix `MoE_ForecasterEnsemble` properties (Issue #48)

### PR DESCRIPTION
**Pull request by isenilov**
This fix addresses the issue https://github.com/salesforce/Merlion/issues/48

### CODE REVIEW

**Comment by isenilov** [comment]
Hi @aadyotb please let me know if this change makes sense or there is another way to tackle the problem. Thanks in advance and Happy Holidays!

**Comment by aadyotb** [comment]
This makes sense, I'll give it an official review after the holidays.
Thanks for the contribution!

File: merlion/models/ensemble/MoE_forecast.py [hunk 1]
```diff
@@ -285,9 +287,6 @@ def moe_model(self, moe_model):
                 self.optimiser = torch.optim.Adam(self.moe_model.parameters(), lr=self.lr, weight_decay=0.00000)
             if self.lr_sch is None:
                 self.lr_sch = LRScheduler(lr_i=0.0000, lr_f=self.lr, nsteps=self.warmup_steps, optimizer=self.optimiser)
-        else:
-            self.optimiser = None
-            self.lr_sch = None
```
Comment (aadyotb) [inline merlion/models/ensemble/MoE_forecast.py:290]: Can you add back these lines? I think this is an important check to ensure that the optimizer and LR scheduler remain consistent in case the MoE model is set to `None`.

**Review by aadyotb — CHANGES_REQUESTED** [review]
Thanks for the fix! I left a small comment for you to address, but otherwise this looks good to me.

File: merlion/models/ensemble/MoE_forecast.py (hunk 1 above)
Comment (isenilov) [reply merlion/models/ensemble/MoE_forecast.py:290]: Sure, will do. Thanks for the explanation, it just felt redundant within the local context.

**Review by aadyotb — APPROVED** [review]
